client/screens/feed/Feed.js

```diff
@@ -15,13 +15,7 @@ import {
   Flex,
 } from 'native-base';
 import { AntDesign } from '@expo/vector-icons';
-import {
-  StyleSheet,
-  FlatList,
-  View,
-  ScrollView,
-  RefreshControl,
```

Comment:
Why removing this ?

Reply:
didnt remove this just due to identation changes its showing its their